index.bs

```diff
@@ -3406,6 +3421,68 @@ and |session|:
 
 </div>
 
+<div algorithm="collect nodes using accessibility attributes">
+To <dfn>collect nodes using accessibility attributes</dfn> with given |context nodes|, |selector|,
+|maximum returned node count|, and |returned nodes|:
+
+1. If |returned nodes| is null:
+
+  1. Set |returned nodes| to an empty [=/list=].
+
+1. For each |context node| in |context nodes|:
+
+  1. Let |match| be true.
```

Comment:
Should this by default really be true? It would mean that if a selector is used that doesn't contain any `name` or `role` we would match all the nodes in the document. This could be kinda invasive for huge documents. Or can a computed role / name never be `undefined`?

Nevertheless isn't it better to set it to `false` here by default and to `true` only when a match has found?

Reply:
That is checked in the beginning of the main algorithm, so at least one attribute will be provided. Setting default to false makes the text more verbose as it cannot be flipped to true based on a single attribute check.